authority/src/lib.rs

```diff
@@ -342,7 +342,11 @@ pub mod module {
 			Ok(())
 		}
 
-		#[pallet::weight(T::WeightInfo::authorize_call())]
+		#[pallet::weight((
+			T::WeightInfo::authorize_call(),
+			DispatchClass::Operational,
+			Pays::No,
```

Comment:
I am not 100% sure if this will still charge fee if this call failed. Can you verify?

Reply:
ok

Reply:
I know it is safe to do like this https://github.com/open-web3-stack/open-runtime-module-library/blob/69426e8597ebc7c7922b81dbb2f6cc2ab7851e86/oracle/src/lib.rs#L165
An error will not return Pays::No.

Reply:
@xlc yes it will

Reply:
Post dispatch info will be `{ post_info: PostDispatchInfo { actual_weight: None, pays_fee: Pays::Yes }`